queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+vet_clinic=# SELECT * FROM animals WHERE name LIKE '%mon';
```

Comment:
- [ ] Hello, in this file, let's only submit the SQL query. Not as it had been in the terminal. Instead of 👇🏼:

  ```SQL
  vet_clinic=# SELECT * FROM animals WHERE name LIKE '%mon';
  ```
  
  Let's do 👇🏼:
  
  ```SQL
  SELECT * FROM animals WHERE name LIKE '%mon';
  ```
  
  Please refactor other queries in this file following this given example.